flow-typed/buildo.js

```diff
@@ -0,0 +1,15 @@
+type Many<T> = T | T[];
```

Comment:
discussing offline:
- this file is for "missing typings", where missing could be either:
  - not part of the package on npm or in any typing repo (applies both to Flow and TypeScript) (the case of `tcomb-react`)
  - incomplete/loose typing available, or, like it is the case for the `lodash` one here: we'd want to limit the `omit` and `pick` apis to something more strict and less funky JS (e.g. required first Object and/or required at least something to "pick"/"omit")
- it is named "buildo" because in theory, such typings could be shared cross-repo, but this is a whole new field to explore

Summing up, this file should probably be renamed (and maybe split), and we should keep just what we deem to be strictly needed to support development (with Flow)